test/lit.site.cfg.in

```diff
@@ -88,6 +88,9 @@ config.available_features.add("CMAKE_GENERATOR=@CMAKE_GENERATOR@")
 if "@SWIFT_ENABLE_SOURCEKIT_TESTS@" == "TRUE":
     config.available_features.add('sourcekit')
 
+if "@SWIFT_BUILD_PERF_TESTSUITE@" == "TRUE":
+    config.available_features.add('benchmark')
```

Comment:
I'm sorry for that! But I'm not sure how to properly fix that. I've added this condition following [explicit guidance](https://github.com/apple/swift/pull/12415#issuecomment-404262856) from @eeckstein. Any pointers for analogous dynamic check?

Reply:
Hm, well, you're already using `inferSwiftBinary` to check for the presence of `Benchmark_O`. Maybe you can see if that came back negative? (meaning, it returned the unqualified `Benchmark_O` instead of finding the actual executable)

Reply:
I'm a bit tick today… so let me walk through this slowly, if I get your meaning:
All these `lit.cfg` files look like configuration files, but are in fact Python scripts. Which means I can extend that offending `if` with additional check for `config.benchmark_o` which carries the result from `inferSwiftBinary` and I check that it is not  just plain `'Benchmark_O'`. Correct?

Reply:
Yeah, basically. I actually think you should drop the current check and instead put one in the regular `lit.cfg`. IIRC the generated `lit.site.cfg` gets read very early on and shouldn't be doing much other than checking the configuration options.